package.json

```diff
@@ -1,57 +1,38 @@
 {
   "name": "formidable-oss-landers",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.3.2",
-    "@testing-library/user-event": "^7.1.2",
-    "formidable-oss-badges": "^0.3.5",
+  "version": "0.0.1",
+  "main": "dist/formidable-oss-landers.cjs.js",
```

Comment:
For `main` and `module` are consumers typically going to consume _every_ code file we build? Not the biggest deal, but especially for Node.js it's typically nicer to have "real" files (in our case babel-ified individual files).

Reply:
@ryan-roemer do you know if it's possible to achieve this with rollup? I just followed their [starter template](https://github.com/rollup/rollup-starter-lib) but it seems like we might have to use webpack to achieve this. Or just use babel directly outside of rollup maybe?

Reply:
I think just babel directly outside of rollup should be fine. And, if we're doing that we could consider both CJS (mapping to `package.json:main`) and ESM (mapping to `package.module`) outputs of individual files like we do for lots of our libraries already.